subsys/net/lib/http/http_server_core.c

```diff
@@ -12,6 +12,8 @@
 #include <string.h>
```

Comment:
Could you fix the commit subject of `lib: net: http: add missing includes` to `net: lib: http: add missing includes`. We try to keep the `net:` prefix for network related commits.